testing/armeria-shaded-for-testing/build.gradle.kts

```diff
@@ -28,4 +25,14 @@ tasks {
     relocate("META-INF/native/netty", "META-INF/native/io_opentelemetry_testing_internal_netty")
     mergeServiceFiles()
   }
-}
\ No newline at end of file
+
+  val extractShadowJar by registering(Copy::class) {
```

Comment:
I now propose to stop and take a step back. Extracting shadow jar that we have just built seems like we are doing something very-very wrong.